### PR TITLE
Reworking project charter to include a TOC and remove the PMCC/PMC

### PR DESCRIPTION
**Pull request by chipchilders**
PR to use to evaluate proposed project changes. This is based on the work of the technical community to restructure their own governance processes. This PR will not be merged until the Governing Board approves this change.

### CODE REVIEW

File: governing-board/charter.md [hunk 1]
```diff
@@ -157,6 +230,8 @@ c)	The Governing Board manages the CFF. The CFF will also have an Outreach Commi
 
 -   d)	In the event of a tied vote with respect to an action that cannot be resolved by the Governing Board, any member of the Governing Board may refer the matter to the LF for assistance in helping the Governing Board reach a decision. If there is a tied vote in any Committee that cannot be resolved, the matter may be referred to the Governing Board.
 
+-   e)  This section (Section 8) does not apply to the TOC. TOC decision making rules are established in Section 7.d.
+
```
Comment (loewenstein) [inline governing-board/charter.md:234]: Wouldn't it be a good idea to align decision making in all governing bodies of the CFF?

**Review by loewenstein — CHANGES_REQUESTED** [review]

File: governing-board/charter.md [hunk 2]
```diff
@@ -111,41 +111,114 @@ c)	The Governing Board manages the CFF. The CFF will also have an Outreach Commi
...
+    -   iv) The TOC shall record calls to vote and decisions voted upon, including the individual votes of the TOC members, in the regularly published TOC meeting notes.
+
+-   e)  TOC Elections
+
+    -   i) Candidate Eligibility
+
+        -   (1)  Current TOC members, Working Group Leads, and Approvers with at least 3 months tenure are eligible to stand for election.
+        
+        -   (2)  Candidates may self-nominate or be nominated by another eligible member.
```
[the remark this answers is not in the document]
Comment (chipchilders) [reply governing-board/charter.md:199]: Links can be added to the non-normative copy (this GH repo) at a later date. The "official" version of this document will live within the foundation's business governance legal document system.

File: governing-board/charter.md (hunk 1 above)
Comment (chipchilders) [reply governing-board/charter.md:234]: As discussed last week, business and technical governance are distinct from each other, by design. What works for the GB, legal committee (if formed) outreach committee, etc... is not necessarily the same as the TOC.

File: governing-board/charter.md [hunk 3]
```diff
@@ -111,41 +111,114 @@ c)	The Governing Board manages the CFF. The CFF will also have an Outreach Commi
...
+
+-   b)	Composition:
+
+    -   i)	The TOC will be composed of five (5) members.
+
+    -   ii) Each TOC member will be elected to a two-year term.
+
+    -   iii) There will be an annual election to determine the composition of the TOC for the following year. Three seats will be up for election in one year and two will be up for election the following year.
+
+    -   iv) Employees from the same company or Related Companies (as defined in Section 9) should not hold more than two TOC seats.
```
[the remark this answers is not in the document]
Comment (chipchilders) [reply governing-board/charter.md:163]: At one point during the process of drafting, we discussed how the rules establishing the TOC should account for situations where perhaps there is not enough diversity in the community to make this a MUST statement. Is this a change that your legal team needs to see made?

[the remark this answers is not in the document]
Comment (loewenstein) [reply governing-board/charter.md:163]: I'd argue that b) below is enough to define this specific exception.

**Comment by chipchilders** [comment]
Merging the changes that were approved by the Governing Board.

**Pull request merged**